Frameworks/CoreFoundation/NumberDate.subproj/CFTimeZone.c

```diff
@@ -53,26 +55,12 @@
 #include <Foundation/NSTimeZone.h>
 #include <Foundation/NSDate.h>
 
-struct tzhead {
```

Comment:
Sounds good to me!

---
In reply to: [72719013](https://github.com/Microsoft/WinObjC/pull/701#discussion_r72719013) [](ancestors = 72719013,72718150)